src/webots/core/WbStandardPaths.cpp

```diff
@@ -232,6 +241,11 @@ const QString &WbStandardPaths::webotsTmpPath() {
         WbLog::fatal(QObject::tr("Cannot create a directory in the Webots temporary directory \"%1\"").arg(webotsTmpPath));
 #endif
     }
+
+    // write a new live.txt file in the webots tmp folder every hour to prevent any other webots process to delete it
+    static QTimer timer;
+    QTimer::connect(&timer, &QTimer::timeout, liveWebotsTmpPath);
+    timer.start(3600 * 1000);
```

Comment:
I would update it every 30min, having the exact same update and time threshold seems dangerous.